sdk/lib/rtl/rtl_vista.h

```diff
@@ -17,4 +17,9 @@
 /* Main RTL Header */
 #include "rtl.h"
 
+#define TRACE(fmt, ...)
+#define WARN(fmt, ...)
+#define FIXME(fmt, ...)
+#define ERR(fmt, ...)
```

Comment:
Should these use `DPRINT*()` or the like? And maybe `#define NDEBUG` where relevant?

Reply:
Ideally we should have a special NT/wine glue header that does what you suggest, together with the other things Timo had to do (`#define GetProcessHeap() RtlGetProcessHeap()` and the like).